.storybook/stories/DragControls.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { withKnobs, number, boolean } from '@storybook/addon-knobs'
+import { DragControls as DragControlsImpl } from 'three-stdlib'
+import { Setup } from '../Setup'
+import { Event, Object3D } from 'three'
+
+import { Box, DragControls } from '../../src'
+
+function DragControlsStory() {
+  const meshRef1 = React.useRef<Object3D>(null!)
+  const meshRef2 = React.useRef<Object3D>(null!)
+  const dragControls = React.useRef<DragControlsImpl>(null!)
+
+  const [objects, setObjects] = React.useState<Object3D[]>()
+
+  React.useEffect(() => {
+    if (!objects) setObjects([meshRef1.current, meshRef2.current])
```

Comment:
If it were a HOC, you could potentially make this redundant.

Reply:
Yes this is quite an akward API compared to the other `drei` components. I will try to find a cleaner solution.